pyecharts/utils.py

```diff
@@ -55,3 +55,50 @@ def json_dumps(data, indent=0):
     :param indent: 缩进量
     """
     return json.dumps(data, indent=indent, cls=UnknownTypeEncoder)
+
+
+def to_css_length(x):
+    """
+    Return the standard length string of css.
+    It's compatible with number values in old versions.
+    :param x:
+    :return:
+    """
+    if isinstance(x, (int, float)):
+        return '{}px'.format(x)
+    else:
+        return x
+
+
+def merge_js_dependencies(*chart_or_name_list):
+    """
+    Merge multiple dependencies to a total list.
+    This will ensure the order and unique in the items.
+    :param chart_or_name_list:
+    :return: A list containing dependency items.
+    """
+    front_must_items = ['echarts']  # items which must be included.
+    front_optional_items = ['echartsgl']
+    dependencies = []
+    fist_items = set()
+
+    def _add(_d):
```

Comment:
this embedded function should be moved outside so that you can test this function too. Since _add is a simple, usually, you may want to rename it to reflects its context: where is it used in its function name. Well you could write a bit of doc string but it's best to "melt" the documentation in its function name.